spec/factories/seeds/induction_programme_factory.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory(:seed_induction_programme, class: "InductionProgramme") do
+    training_programme { "full_induction_programme" }
+
+    trait(:fip) { training_programme { "full_induction_programme" } }
+    trait(:cip) { training_programme { "core_induction_programme" } }
+    trait(:design_our_own) { training_programme { "design_our_own" } }
+    trait(:school_funnded_fip) { training_programme { "school_funnded_fip" } }
+
+    trait(:with_school_cohort) { association(:school_cohort, factory: :seed_school_cohort) }
+    trait(:with_school) { association(:school, factory: :seed_school) }
```

Comment:
✂️

Reply:
But `TeacherProfile` `has_one: :mentor_profile`? That surprised me too!

Reply:
Ouch, I think there are a few crappy associations/scopes that could do with sanitising